### PR TITLE
Fix typo in smart contract security docs

### PR DESCRIPTION
**Pull request by BlockchainRev**
"in the $1B" fixed to "over $1B"

### CODE REVIEW

File: src/content/developers/docs/smart-contracts/security/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ lang: en
...
+Ethereum smart contracts are extremely flexible, capable of both holding large quantities of tokens (often in excess of $1B) and running immutable logic based on previously deployed smart contract code. While this has created a vibrant and creative ecosystem of trustless, interconnected smart contracts, it is also the perfect ecosystem to attract attackers looking to profit by exploiting vulnerabilities in smart contracts and unexpected behavior in Ethereum. Smart contract code _usually_ cannot be changed to patch security flaws, assets that have been stolen from smart contracts are irrecoverable, and stolen assets are extremely difficult to track. The total of amount of value stolen or lost due to smart contract issues is easily over $1B. Some of the larger due to smart contract coding errors include:
```
Comment (minimalsm) [inline src/content/developers/docs/smart-contracts/security/index.md:8]: And then some 😅

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
@BlockchainRev thanks for the PR on this ❤️ separate from this PR I think we should find a more accurate number (and probably a source?) for this information.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @BlockchainRev for docs

**Comment by BlockchainRev** [comment]
Will find a more updated stat. Thanks.

**Pull request merged**